examples/guestbook/README.md

```diff
@@ -539,7 +539,7 @@ spec:
     spec:
       containers:
       - name: php-redis
-        image: gcr.io/google_samples/gb-frontend:v3
+        image: gcr.io/google-samples/gb-frontend:v4
```

Comment:
The hyphen version used to not be properly supported by gcr, but now it is.  They both are fine now.
(the project itself is named google-samples-- but we used to have to use the underscore).